homeassistant/components/fibaro/config_flow.py

```diff
@@ -6,6 +6,7 @@
 from typing import Any
 
 from slugify import slugify
+from url_normalize import url_normalize
```

Comment:
Is this required by the fibaro library? Otherwise we need to add it to the requirements in the manifest.json.

Reply:
Sorry I didn't realize that this is not from the standard lib. I just removed that part, I think this small feature is not worth to add another dependency.